scitbx/array_family/boost_python/flex_vec2_double.cpp

```diff
@@ -256,6 +287,28 @@ namespace {
     return std::sqrt(sum_sq_(self));
   }
 
+  af::shared<vec2<double> >
+  each_normalize(
```

Comment:
I looked at this and thought `each_normalize` is an odd (non-obvious) name to use - straight `normalize` would probably be the name and behaviour I would expect. However I see this was what Ralf chose to use in 2009 for `flex.vec3_double`

I wonder though whether the method name here _should_ be `normalize` and we add an alias for `vec3` & deprecate the old name.

Reply:
Best guess is such a change may not be too egregious, as it is a little used function 

```
silver-surfer cctbx_project :) [master] $ git grep each_normalize .|wc -l
      14
```